From f551ef1599fa4ff5d244285eef731928a5d9820e Mon Sep 17 00:00:00 2001
From: Jonathan Lebon <jlebon@redhat.com>
Date: Tue, 10 Dec 2013 11:38:39 -0500
Subject: BZ1054954: PR16309: spawn stap-serverd with / cwd

---
 stap-start-server | 1 +
 1 file changed, 1 insertion(+)

diff --git a/stap-start-server b/stap-start-server
index b6ec943..4221631 100755
--- a/stap-start-server
+++ b/stap-start-server
@@ -18,6 +18,7 @@
 startup_timeout=10
 
 # start the server
+cd / # change to a dir we're 100% sure we have RD_ONLY access to
 ${stap_pkglibexecdir}stap-serverd "$@" </dev/null >/dev/null 2>/dev/null &
 server_pid=$!
 
-- 
1.8.3.1

