From b5c84307cd507c5fa8d7991e1b533d6983b07a73 Mon Sep 17 00:00:00 2001
From: Jonathan Lebon <jlebon@redhat.com>
Date: Mon, 10 Feb 2014 10:21:26 -0500
Subject: [PATCH] autoreconf

---
 configure       | 3 ++-
 man/Makefile.in | 8 +++++---
 2 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/configure b/configure
index 69231af..2a964c1 100755
--- a/configure
+++ b/configure
@@ -11420,7 +11420,7 @@ STAP_PREFIX="$stap_prefix"
 
 ac_config_headers="$ac_config_headers config.h:config.in"
 
-ac_config_files="$ac_config_files Makefile doc/Makefile man/Makefile doc/beginners/Makefile doc/SystemTap_Tapset_Reference/Makefile man/stappaths.7 initscript/config.systemtap initscript/config.stap-server initscript/systemtap initscript/stap-server initscript/99stap/module-setup.sh"
+ac_config_files="$ac_config_files Makefile doc/Makefile man/Makefile doc/beginners/Makefile doc/SystemTap_Tapset_Reference/Makefile man/stappaths.7 man/systemtap.8 initscript/config.systemtap initscript/config.stap-server initscript/systemtap initscript/stap-server initscript/99stap/module-setup.sh"
 
 
 
@@ -12292,6 +12292,7 @@ do
     "doc/beginners/Makefile") CONFIG_FILES="$CONFIG_FILES doc/beginners/Makefile" ;;
     "doc/SystemTap_Tapset_Reference/Makefile") CONFIG_FILES="$CONFIG_FILES doc/SystemTap_Tapset_Reference/Makefile" ;;
     "man/stappaths.7") CONFIG_FILES="$CONFIG_FILES man/stappaths.7" ;;
+    "man/systemtap.8") CONFIG_FILES="$CONFIG_FILES man/systemtap.8" ;;
     "initscript/config.systemtap") CONFIG_FILES="$CONFIG_FILES initscript/config.systemtap" ;;
     "initscript/config.stap-server") CONFIG_FILES="$CONFIG_FILES initscript/config.stap-server" ;;
     "initscript/systemtap") CONFIG_FILES="$CONFIG_FILES initscript/systemtap" ;;
diff --git a/man/Makefile.in b/man/Makefile.in
index 42c56be..ae68814 100644
--- a/man/Makefile.in
+++ b/man/Makefile.in
@@ -85,7 +85,7 @@ target_triplet = @target@
 @HAVE_LIBVIRT_TRUE@@HAVE_LIBXML2_TRUE@am__append_3 = stapvirt.1
 subdir = man
 DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/Makefile.am \
-	$(srcdir)/stappaths.7.in
+	$(srcdir)/stappaths.7.in $(srcdir)/systemtap.8.in
 ACLOCAL_M4 = $(top_srcdir)/aclocal.m4
 am__aclocal_m4_deps = $(top_srcdir)/m4/gettext.m4 \
 	$(top_srcdir)/m4/iconv.m4 $(top_srcdir)/m4/lib-ld.m4 \
@@ -96,7 +96,7 @@ am__configure_deps = $(am__aclocal_m4_deps) $(CONFIGURE_DEPENDENCIES) \
 	$(ACLOCAL_M4)
 mkinstalldirs = $(install_sh) -d
 CONFIG_HEADER = $(top_builddir)/config.h
-CONFIG_CLEAN_FILES = stappaths.7
+CONFIG_CLEAN_FILES = stappaths.7 systemtap.8
 CONFIG_CLEAN_VPATH_FILES =
 AM_V_P = $(am__v_P_@AM_V@)
 am__v_P_ = $(am__v_P_@AM_DEFAULT_V@)
@@ -321,7 +321,7 @@ top_srcdir = @top_srcdir@
 AUTOMAKE_OPTIONS = no-dist foreign
 man_MANS = stapprobes.3stap stapfuncs.3stap stapvars.3stap \
 	stapex.3stap dtrace.1 stap-merge.1 stappaths.7 stapsh.8 \
-	$(am__append_1) $(am__append_2) $(am__append_3)
+	systemtap.8 $(am__append_1) $(am__append_2) $(am__append_3)
 all: all-am
 
 .SUFFIXES:
@@ -357,6 +357,8 @@ $(ACLOCAL_M4): @MAINTAINER_MODE_TRUE@ $(am__aclocal_m4_deps)
 $(am__aclocal_m4_deps):
 stappaths.7: $(top_builddir)/config.status $(srcdir)/stappaths.7.in
 	cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@
+systemtap.8: $(top_builddir)/config.status $(srcdir)/systemtap.8.in
+	cd $(top_builddir) && $(SHELL) ./config.status $(subdir)/$@
 install-man1: $(man_MANS)
 	@$(NORMAL_INSTALL)
 	@list1=''; \
-- 
1.8.3.1

